packages/core/src/tracing/spanHandle.js

```diff
@@ -165,22 +182,35 @@ SpanHandle.prototype.end = function end(errorCount) {
 };
 
 /**
+ * TODO: make it as a class
```

Comment:
Yes, we should. Is this in the scope for this PR or something for later?

Reply:
I would leave it to another PR, so we keep the context of typing. Even though there were some code changes, but still they were related to the typing.